### ISSUE TITLE
[issue]: 

### ISSUE THREAD

**Issue by frenchiveruti** [ventoy/Ventoy#2884]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.99

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

Both

### Partition Style

MBR

### Disk Capacity

16

### Disk Manufacturer

HP

### Image file checksum (if applicable)

None

### What happened?

Hello, so I'm having a hell of weird issue with Ventoy. 

I have an USB drive (HP v165w 16GB) to which I installed Ventoy on a Windows10 machine [using the Ventoy2Disk app following this simple guide](https://ventoy.net/en/doc_start.html).   

The Motherboard model is [Gigabyte B85M-D3H-A](https://www.gigabyte.com/Motherboard/GA-B85M-D3H-A-rev-10#ov)

I have tried all these configs:

1. exFAT/GPT/SecureBoot
2. exFAT/MBR/SecureBoot
3. FAT32/GPT/SecureBoot
4. FAT32/MBR/SecureBoot
5. FAT32/MBR/NON SecureBoot
6. FAT32/GPT/NON SecureBoot

I also tried all the options on the UEFI; Legacy, UEFI, Legacy & UEFI, UEFI first, Legacy First, in terms of combinations I did not go all the way as it's a gigantic matrix of options (combined with the aforementioned Ventoy configs).  

The ISO files I'm trying to use are:

1. OpenMediaVault (700MB)
2. Lubuntu 24 (3.04GB)

&nbsp;  

Now, what I **did find** is that when I have the VENTOY (F:) partition on windows **empty** the BIOS **detects** the USB and I can even launch Ventoy, of course it says it has no ISO files. **It is launching as 1.0.99 UEFI (Bottom left corner)**

Now, whenever I load an ISO file in the VENTOY(F:) drive, the USB is **no longer detected**, which is absolutely weird. 

One thing I noticed, using MiniTool Partition Wizard, is that the Ventoy EFI partition is set to the "end" of the drive, meaning that the VENTOY(F:) partiton is first, and then after that I have the 32MB VTOYEFI partition that holds Ventoy itself.  

&nbsp;

What is going on? I tried google for a while but couldn't find anything similar to my issue. 

#TL;DR: If I place an ISO on the VENTOY USB drive, it stops being detected by the BIOS.

**Comment by steve6375** [comment]
HP v165w 16GB is probably a fake USB drive (fake capacity)
The first say 4GB may exist, but the other 12GB are probably non-existent and the space is mapped to the first 4GB.
When you add an ISO, it writes to the first 4GB and overwrites the small Ventoy Ptn 2.
Try a different USB flash drive or test it with FakeFlashTest.exe or Validrive.exe

**Comment by frenchiveruti** [comment]
> HP v165w 16GB is probably a fake USB drive (fake capacity) The first say 4GB may exist, but the other 12GB are probably non-existent and the space is mapped to the first 4GB. When you add an ISO, it writes to the first 4GB and overwrites the small Ventoy Ptn 2. Try a different USB flash drive or test it with FakeFlashTest.exe or Validrive.exe

Hello, although it's a good take on the issue, I've used this USB countless times for many different uses, it's for sure not fake.   
Furthermore, when I **remove** the file from the ISO, Ventoy appears again in the boot list, therefore it's not being overwritten

**Comment by steve6375** [comment]
Simple test - try a different USB drive to see if a different drive behaves in the same way.
Try another computer/notebook
You could also try a VM or emulator. 
MobaLiveCD and RMPrepUSB both use QEMU which you can use to boot from the Ventoy USB drive under Windows

P.S. Just because you have used it many times does not mean that it isn't fake.
e.g.
My gallon bucket has a hole in the side about 3/4 of the way up. I have used it hundreds of times and it will hold 4 pints of water and never leaks so it must be good!

**Comment by Dany0x0** [comment]
Actuall its true, there is some sort of bug in ventoy (latest version) , due to which it isn't working on hp usb, my hp usb (2.0/3.0) is getting dismounted every time i install ventoy.
I t does work on other usb like sand disk however it keeps messing up in hp usb, i tried it on my friends laptop, and there was the same issue.
Hope ventoy fixes it in next update.

**Comment by frenchiveruti** [comment]
> Simple test - try a different USB drive to see if a different drive behaves in the same way. Try another computer/notebook You could also try a VM or emulator. MobaLiveCD and RMPrepUSB both use QEMU which you can use to boot from the Ventoy USB drive under Windows
> 
> P.S. Just because you have used it many times does not mean that it isn't fake. e.g. My gallon bucket has a hole in the side about 3/4 of the way up. I have used it hundreds of times and it will hold 4 pints of water and never leaks so it must be good!

Dude, fine, the world is filled with morons but I ain't one that comes around and puts 3h trying to figure what is wrong with Ventoy and why is behaving like this, without covering the bases. 

But, given that you have mentioned several times and insisted on the "Your USB is fake and you never even used it full" (I did); I ran the FAKEFLASHTEST v1.1.2 and this is the result:
![image](https://github.com/ventoy/Ventoy/assets/1095997/6e026a93-a333-474c-9b28-1a8589eb0f29)

As you can see, **it's not fake**, the device is fine, it's ventoy that has some hiccup on it. 

I ended up following your suggestion and used a sandisk cruzer glide 32gb and ventoy with ISOs on the partition was detected fine. But the HP issue, still remains. Hence I'll leave this open.

Bests,

**Comment by Merconix** [comment]
Just experienced the same issue on my SanDisk USB 3.2 Gen 1 64GB flash drive as well.
Previous versions worked fine, then I installed the ARM version of 1.99 which also worked fine, but upon installing the x86/64 version onto my flash drive it would simply hang every boot before displaying anything.

Multiple reinstalls, upgrades, etc. didn't solve the issue, it would not boot on any device I plugged it into.

Using the "upgrade" feature of the installer for Ventoy 1.75 (just the older version I had on hand) got me a stick that works just like it used to, so it's not an issue with the stick or my machines.

**Comment by ElryGH** [comment]
Same issue here, i hate people like steve, useless people like you shouldnt be on github

**Comment by steve6375** [comment]
I'm only trying to help. I suggested using a different flash drive and that worked. So I have helped haven't I?
No one else has tried to help you guys and I have spared my time to try to help you. I don't know how experienced you are or what else you have tried other than what you tell me. So fine, from now on I won't try to help you with my 30 years of professional experience in computer development. Bye

**Comment by ElryGH** [comment]
Noone needs your "professional help" if all you do is say it cant be true and that the issue is a fake usb, bye.

**Comment by frenchiveruti** [comment]
> I'm only trying to help. I suggested using a different flash drive and that worked. So I have helped haven't I? No one else has tried to help you guys and I have spared my time to try to help you. I don't know how experienced you are or what else you have tried other than what you tell me. So fine, from now on I won't try to help you with my 30 years of professional experience in computer development. Bye

Hey steve although I don't agree with your methods you do help. 
Just wanted to say that, we could've spent days trying to figure it out when the easiest solution was to use another USB drive. 

The only comment, just keep in mind that folks sometimes are unable to afford a second USB drive. 

Thank you for your help!
